src/tests/JIT/Regression/JitBlue/Runtime_62692/Runtime_62692.cs

```diff
@@ -39,8 +40,16 @@ public static int TestEntryPoint()
             AssertEqual(Problem2(1111, 0xFFFF_FFFF_0000_0001), 3414328792);
             AssertEqual(Problem3(1, 0xFFFF_0001), 0);
             AssertEqual(Problem4(1111, 0xFFFF_FFFF_0000_0001), 3414328792);
-            AssertEqual(Problem5(1111, double.MaxValue), 3307008522);
-            AssertEqual(Problem6(1111, float.MaxValue), 3307008522);
+            if (RuntimeInformation.ProcessArchitecture == Architecture.X64 || RuntimeInformation.ProcessArchitecture == Architecture.X86)
```

Comment:
Arm64 should have the desired saturating behavior already and that this PR should switching x86/x64 to. Why is existing test passing on Arm64?

Reply:
I think this is some other issue. I need to check if Arm64 even runs this set of tests. Because if Arm use to run this tests and we know that it already has saturating behavior, then they would be failing on X86/X64 (since they do not saturate). but they are not failing. This tells me that there is a chance that arm never runs this test. Looking into it.

Reply:
Right, there is `Sse42.IsSupported` condition a few lines above that implies x86/x64. So this only ever runs on x86/x64 and this if condition is not needed.

Reply:
Removing the if condition. Will make this changes today.

Reply:
Pushed this change.